docs/lint_errors.md

```diff
@@ -122,58 +122,61 @@ nf-core pipelines must have CI testing with GitHub Actions.
 
 ### GitHub Actions
 
-There are 3 main GitHub Actions CI test files: `ci.yml`, `linting.yml` and `branch.yml` and they can all be found in the `.github/workflows/` directory.  
+There are 3 main GitHub Actions CI test files: `ci.yml`, `linting.yml` and `branch.yml` and they can all be found in the `.github/workflows/` directory.
 
-This test will fail if the following requirements are not met in these files:  
+This test will fail if the following requirements are not met in these files:
 
-1. `ci.yml`: Contains all the commands required to test the pipeline  
-    * Must be turned on for `push` and `pull_request`.  
-    * The minimum Nextflow version specified in the pipeline's `nextflow.config` has to match that defined by `nxf_ver` in this file:  
+1. `ci.yml`: Contains all the commands required to test the pipeline
+    * Must be turned on for `push` and `pull_request`.
```

Comment:
```suggestion
    * Must be turned on for `push` and `pull_request`.
    ```yaml
    on: [push, pull_request]
    ```
```

Reply:
just a small suggestion here

Reply:
I edited your suggestion to add in an extra blank line to save the markdown lint, but yeah - more examples is always good 👍